inspire_disambiguation/__init__.py

```diff
@@ -23,12 +23,37 @@
 """Disambiguation module."""
 import logging
 
+import structlog
+import sys
+from structlog_sentry import SentryJsonProcessor
+
 from .app import BeardConfig  # noqa: F401
 
 conf = BeardConfig()
 
-root = logging.getLogger()
 logging.basicConfig(
-    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-    level=conf["LOG_LEVEL"]
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    stream=sys.stdout,
+    level=conf["LOG_LEVEL"],
+)
```

Comment:
I'd go for format like:
```
format="%(message)s",
```

Reply:
This was just a copy-paste from hep
:+1: